### PR TITLE
Slack edits

### PR DESCRIPTION
**Pull request by kaseybon**
@cromwellryan new slack page is ready for review. Also made some updates to the Services README.

### CODE REVIEW

File: services/README.md [hunk 1]
```diff
@@ -2,15 +2,16 @@ Services
 ========
 
 The services we use. Some we pay for, others we mooch off. We love them all.
-Except Skype.
 
-* **[Dropbox](dropbox)**
 * **[CircleCI](circleci)**
 * **[Code Climate](code_climate)**
-* **[Sprintly](sprintly)**
+* **[Coveralls](coveralls)**
```
Comment (yock) [inline services/README.md]: Thought we were phasing this one out.

**Comment by kaseybon** [comment]
@cromwellryan and @yock Coveralls and Sprintly were removed.

**Comment by cromwellryan** [comment]
Any chance we can add some details about how we use slack then get this closed out monday?

**Comment by kaseybon** [comment]
@cromwellryan The Slack page has been updated with more details. Let me know if there is anything else that should be added.

**Comment by cromwellryan** [comment]
Looks great @kaseybon


Sorry it took so long to pull in.  Fell off my radar :(

**Pull request closed without merging**